docker/docker-compose-monitoring.yml

```diff
@@ -0,0 +1,108 @@
+version: '3.3'
+services:
+
+  prometheus:
+    image: ${DOCKER_HUB_USERNAME}/prometheus:2.37.6
+    ports:
+      - 9090:9090
```

Comment:
FYI: publishing prometheus ports (and other exporters) without authentication is good for testing purposes and bad for production usage. Usually you should place your prometheus behind grafana and publish grafana web interface only. Also make sense do not publish your exporters (mongodb, cadvisor and so on) to the Internet too